### PR TITLE
Add page title and meta description -vnext

### LINKED ISSUE #2390: Update both page title and meta desc of the following LOB samples
- [Boston marathon](https://www.infragistics.com/angular-demos-lob/grid/grid)
- [FinJS ](https://www.infragistics.com/angular-demos-lob/finjs-sample)- page title: Real-Time Financial Analysis Data Grid
- [TreeGrid FinJS
](https://www.infragistics.com/angular-demos-lob/treegrid-finjs-sample)
- [CRM](https://staging.infragistics.com/angular-demos-grid-crm/grid-crm)
- [Data Analysis with dock manager](https://www.infragistics.com/angular-demos-lob/grid-dynamic-chart-data/data-analysis-dock-manager) - page title: Real-Time Data Analysis and Dashboards
- [Data Analysis](https://www.infragistics.com/angular-demos-lob/grid-dynamic-chart-data/data-analysis)
- [Master detail 
](https://www.infragistics.com/angular-demos-lob/grid/grid-master-detail)

All samples should have:
- page title
- meta description

@aatanasov2, could you please assist with the meta desc and page title

### PR DESCRIPTION
**Pull request by dobromirts**
Closes #2390

### CODE REVIEW

**Review by HristoP96 — CHANGES_REQUESTED** [review]
The grid dynamic chart data title and description probably should be switched with the title and description of the finjs component

File: projects/app-lob/src/app/seo.service.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+export class SEOService {
+    private defaultTitle: string = 'Build Better Modern Web Experiences, Faster with Angular Components';
+    private defaultDesc: string = 'Create feature-rich, progressive web apps with Ignite UI for Angular! A complete library for Angular native UI components, including the fastest Angular data grid and 60+ high-performance charts.'
+
+
+    constructor(private title: Title, private meta: Meta) { }
+
+
+    updateTitle(title: string) {
+        title ? this.title.setTitle(title) : this.title.setTitle(this.defaultTitle);
```
Comment (HristoP96) [inline projects/app-lob/src/app/seo.service.ts:17]:
```suggestion
        this.title.setTitle(title ?? this.defaultTitle)
```

File: projects/app-lob/src/app/seo.service.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    }
+
+    updateDescription(desc: string) {
+        desc ? this.meta.updateTag({ name: 'description', content: desc }) : this.meta.updateTag({ name: 'description', content: this.defaultDesc })
```
Comment (HristoP96) [inline projects/app-lob/src/app/seo.service.ts:21]:
```suggestion
        desc ? this.meta.updateTag({ name: 'description', content: desc }) : this.meta.updateTag({ name: 'description', content: this.defaultDesc })
```
```suggestion
        this.meta.updateTag({ name: 'description', content: desc ?? this.defaultDesc });
```

**Review by HristoP96 — CHANGES_REQUESTED** [review]

**Review by HristoP96 — CHANGES_REQUESTED** [review]
Requested changes are commented [here](https://github.com/IgniteUI/igniteui-angular-samples/pull/2396#pullrequestreview-546191167)

**Review by HristoP96 — APPROVED** [review]

**Review by zdrawku — APPROVED** [review]

**Pull request merged**